openc3-cosmos-init/plugins/packages/openc3-cosmos-demo/targets/INST2/cmd_tlm/_ccsds_cmd.txt

```diff
@@ -1,7 +1,7 @@
   PARAMETER    CCSDSVER        0   3  UINT  0     0   0 "CCSDS primary header version number"
   PARAMETER    CCSDSTYPE       3   1  UINT  1     1   1 "CCSDS primary header packet type"
   PARAMETER    CCSDSSHF        4   1  UINT  0     0   0 "CCSDS primary header secondary header flag"
-  ID_PARAMETER CCSDSAPID       5  11  UINT  0  2047 999 "CCSDS primary header application id"
+  ID_PARAMETER CCSDSAPID       5  11  UINT  0  2047 <%= id %> "CCSDS primary header application id"
```

Comment:
It's weird that we hard coded this in the demo but the PKTID was always covering it. If you're going to do it here you should do it in INST as well.